internal/relay/relayer.go

```diff
@@ -80,6 +80,7 @@ func (r *Relayer) Run(
 			if err != nil {
 				r.logger.Error("could not process message", zap.Uint64("query_id", query.Id), zap.Error(err))
 				neutronmetrics.AddFailedRequest(string(query.QueryType), time.Since(start).Seconds())
+				return err
```

Comment:
Looks like we stop the app in case of any error in the `processMessageKV` and `processMessageTX`, regardless is the error happends during tx submition(we want to stop), or during some KV query on the remote chain rpc(we dont want to stop). We need to detect the error is critical and stop only this case

Reply:
@sotnikov-s ok, thanks. added this error into ignore pattern

Reply:
@swelf19 fixed